packages/terra-form-input/src/terra-dev-site/doc/example/controlled/DefaultExample.jsx

```diff
@@ -17,12 +17,12 @@ class DefaultExample extends React.Component {
   render() {
     return (
       <div>
-        <Field label="Default" htmlFor="default-controlled">
+        <Field label="Default">
           <Input
             name="default input"
-            id="default-controlled"
             value={this.state.input}
             onChange={this.handleChange}
+            ariaLabel="form-input-label"
```

Comment:
Let's update this to the following:
```suggestion
            ariaLabel="Default"
```